STEER/Utilities/alihadd.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+# source $ALICE_ROOT/libexec/alihadd.sh
+# Example usage:
+#   source $ALICE_ROOT/libexec/alihadd.sh
+#    * Filter input list before merging
+#       makeFilteredList input.list highPt filter.log
+#    *  Internal function
+#       testFileList input.list highPt | tee  filter.log
+
+
+loadUtilities(){
+    #
+    #  load utilities if not yet done
+    if [ -z `type -t alilog_info` ]; then
+        source $ALICE_ROOT/libexec/alilog4bash.sh
+        alilog_info "Load ALICE_ROOT/libexec/alilog4bash.sh"
+    fi;
+    if [  -z `type -t guessPeriod` ]; then
+        source $ALICE_ROOT/libexec/utilities.sh
+        alilog_info "Load $ALICE_ROOT/libexec/utilities.sh"
+    fi;
+}
+
+testFileList(){
+    # Internal function: filter input list
+    # Provide good/bad file list out of inputList(param[$1])
+    # Tests: read/write/delete/eventually copytree for selected keys(param[$2])
+    inputList=$1
```

Comment:
Until now I was using lower case variables.
In case you prefer some convention, I suggest to write list of suggestion 
per bash scripting and bash testing.
Short text document should be sufficient

Usually when I did not know how to implement something  - I consulted stack-overflow.
Of course people were using there different styles...